AgilProjekt/src/components/QuizComponent.vue

```diff
@@ -171,6 +169,7 @@ const handleHintPopupVisibility = (visible) => {
   color: white;
   margin-bottom: 3rem;
   font-size: 2.5rem;
+  min-height: 100vh;
```

Comment:
Placing min-height at heading makes just the heading take up alot of space

Reply:
> Placing min-height at heading makes just the heading take up alot of space. It might be better if you add the min-height in .quiz styling

@RehasLiamsi 
 Done! 
Thanks.